nfs-ganesha/build/build_deb

```diff
@@ -3,14 +3,21 @@ set -ex
 
 # Only do actual work when we are a DEB distro
 if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
-    exit 0
+    exit 1
 fi
 
+REPO_URL="https://shaman.ceph.com/api/repos/ceph/$CEPH_BRANCH/$CEPH_SHA1/$DISTRO/$DIST/repo"
+if `curl --fail -L $REPO_URL`; then
+  echo "Ceph debian lib repo exists in shaman"
+else
+  echo "Ceph debian lib repo does NOT exist in shaman"
+  exit 1
+fi
 
 # We need this for system, to build ganesha, and to enable FSAL_CEPH and 
 # FSAL_RGW in the .spec file when cmake command runs
```

Comment:
@ktdreyer some of the packages are needed for the cmake run, others are needed because I don't pass the '-d' flag to dpkg-buildpackage which aborts by default if the build dependencies aren't on the host machine. I can go ahead and add it back in, but I was told it's bad practice. Would you have me change another way?

I'll fix the ".spec file" typo as well!